benchmarks/pipe.py

```diff
@@ -9,9 +9,10 @@
 from torchtext.data.utils import get_tokenizer
```

Comment:
why did you rename the file to pipe.py?

Reply:
I feel like the aspect of Fairscale that it's benchmarking is primarily pipe (though it should also detect regressions in purefp16)